app/components/candidate_interface/previous_applications_component.rb

```diff
@@ -0,0 +1,70 @@
+module CandidateInterface
+  class PreviousApplicationsComponent < ViewComponent::Base
+    include ViewHelper
+
+    def initialize(candidate:)
+      @candidate = candidate
+    end
+
+    def render?
+      application_choices.present?
+    end
+
+    def application_choices
+      (previous_application_choices + eligible_current_application_choices)
+        .compact
+        .sort_by(&:id)
```

Comment:
`
.sort_by(&:id)
.reverse
`

I think the most recently submitted application choices should be at the top.

Reply:
Yep no prob - now sorting by `updated_at` field